UPGRADE-3.4.md

```diff
@@ -269,6 +269,11 @@ SecurityBundle
     as first argument. Not passing it is deprecated and will throw a `TypeError`
     in 4.0.
 
+ * Added `logout_on_user_change` to the firewall options. This config item will
+   trigger a logout when the user has changed. Should be set to true to avoid
+   deprecations in the configuration.
+
+
```

Comment:
extra empty line

Reply:
A note should also be added in the 4.0 CHANGELOG.

Reply:
Just the CHANGELOG? Then I'll leave the 4.0 log as is for the PR that actually removes it

Reply:
> I'll leave the 4.0 upgrade log as is for the PR that actually removes it

please do it now on 3.4 so that the notice is also in 3.4, that's the policy we're always following here.